.rhdh/docs/openshift.md

```diff
@@ -0,0 +1,255 @@
+## Using the Operator to manage Red Hat Developer Hub (RHDH)
+
+### Installing RHDH using Operator
+
+**Prerequisites**
+
+- You are logged in using the OpenShift web console.
+- You have configured the appropriate roles and permissions within your project to create an application.
```

Comment:
Can we specify or add a link to the sufficient documentation for the "appropriate roles and permissions" ?

Reply:
does this happen automatically when you create a project/namespace? 

For example on my clusterbot instance (logged in as `kubeadmin`), I just do this to create a project for my backstage deployment, and then add a pull secret to it so I can pull from quay.io:

```
oc new-project mybackstage
oc create -f /tmp/my_quay_pull_secret -n mybackstage
```

where the secret file is the one provided by Quay, eg.,

```
apiVersion: v1
kind: Secret
metadata:
  name: rhdh-pull-secret
data:
  .dockerconfigjson: your-digest-goes-here=
type: kubernetes.io/dockerconfigjson
```

Reply:
Added link in [`1886938` (#118)](https://github.com/janus-idp/operator/pull/118/commits/1886938d5c790b50f4e4209c724af26afc39ee07)